tests/js/lib/personalization.test.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import { get as getCookie, remove as removeCookie } from 'js-cookie';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getUuidFromVisitorCookie,
+	getVisitorCookie,
+	getVisitorCookieRaw,
+} from '../../../src/js/lib/personalization';
+
+describe( 'lib/personalization', () => {
+	beforeEach( () => {
+		Object.keys( getCookie() ).forEach( ( cookieName ) => removeCookie( cookieName ) );
+	} );
+
+	describe( 'getVisitorCookieRaw', () => {
+		test( 'no set cookies should return undefined', () => {
```

Comment:
The wording of this seems ambiguous - I read that as "no cookies which are set, should return undefined" followed by a test that expects the raw cookie value to be undefined.

Try:
```suggestion
		test( 'no cookies are set, so should return undefined', () => {
```